src/minio/core/minio_client.py

```diff
@@ -81,3 +84,231 @@ async def test_connection(self) -> bool:
         except Exception as e:
             logger.error(f"Connection test failed: {e}")
             return False
+
+    async def create_bucket(self, bucket_name: str) -> bool:
+        """
+        Creates a new bucket on the MinIO server.
+
+        Args:
+            bucket_name: The name of the bucket to create.
+
+        Returns:
+            True if the bucket was created successfully.
+
+        Raises:
+            BucketOperationError: If the bucket creation fails for any reason.
```

Comment:
Not really any point in returning something if there's only ever one possible value

Reply:
yea yea. Removed return.